lib/widget/form_textfield.dart

```diff
@@ -101,7 +101,7 @@ abstract class BaseTextInput extends StatefulWidget with Invokable, HasControlle
       'validator': (value) => _controller.validator = Utils.getValidator(value),
       'obscureToggle': (value) => _controller.obscureToggle = Utils.optionalBool(value),
       'keyboardAction': (value) => _controller.keyboardAction = _getKeyboardAction(value),
-      'maxLines' : (value) => _controller.updateMaxline(value),
+      'maxLines' : (value) =>  _controller.maxLines = Utils.getInt(value, min: 1, fallback: _controller.maxLines),
```

Comment:
@hamxa152 I don't understand. First you said that Utils.getInt(value, min: 1, fallback: _controller.maxLines) is not always working and then you just used it here?

Reply:
when we set int? maxLines then its always set null value when maxlines is not define in ymal file 
but when  int maxLines = 1 then its work fine.

Reply:
so the problem was always with your controller.maxLines as it was an optional int with the default value of null?